src/util/fieldLoader.ts

```diff
@@ -8,34 +8,39 @@ import axios from 'axios';
 // To avoid throwing errors with 404.
 axios.defaults.validateStatus = () => true;
 
-class FieldLoader {
-  /**
-   * Loads the package.json required fields as a JS object (default ../config/fields.json)
-   * @param customPath the path of the custom specified required fields (optional)
-   * @returns the required fields as a JS object
-   */
-  async loadFields(customPath?: string): Promise<Record<string, unknown>> {
-    const location = customPath
-      ? path.join(process.cwd(), customPath)
-      : path.join(__dirname, '..', 'config', 'fields.json');
+/**
+ * Loads the package.json required fields as a JS object (default ../config/fields.json)
+ * @param customPath the path of the custom specified required fields (optional)
+ * @returns the required fields as a JS object
+ */
+async function loadFields(
+  customPath?: string
+): Promise<Record<string, unknown>> {
+  const location = customPath
+    ? path.join(process.cwd(), customPath)
```

Comment:
Pattern: Prefer resolve over join

Reply:
I didn't know that a `resolve` thing existed. Thanks!